README.md

```diff
@@ -21,7 +21,7 @@ The project aims to build a continuous deployment pipeline to offer FLINT as a S
 
 In order to run the REST API, please follow the following steps: - 
 
-1. `docker build -t flint-api .`
+1. `cd local/rest_api_flint.example && docker build -t flint-api .`
 2. `docker run --rm -p 8080:8080 flint-api`
```

Comment:
Hi @khabdrick, I can see these are just terminal commands on their own. Can you help the reader out by explaining what they should enter these commands into (I'm assuming any terminal will do), and in which directory/folder? If you could do the same for lines 38-42 that would also be great!